presto-main/src/test/java/com/facebook/presto/operator/scalar/TestRegexpFunctions.java

```diff
@@ -93,10 +93,10 @@ public void testRegexpReplace()
         assertInvalidFunction("REGEXP_REPLACE('xxx', 'x', '$a')", INVALID_FUNCTION_ARGUMENT);
```

Comment:
tests should execute for both REJ and JONI implementations of Regex functions. Maybe it could be solved by using testNG parametrization with two session objects (one with `regex_library` set to `RE2J` and other with `regex_library` set to `JONI`). We can discuss that.